apps/docs/content/components/alert/usage.ts

```diff
@@ -0,0 +1,19 @@
+const App = `import {Alert} from "@nextui-org/react";
+
+export default function App() {
+  const title = "Email Sent!!";
+  const description ="You will get a reply soon";
+  return (
+    <div className="flex items-center justify-center w-screen">
+          <Alert title = {title} description = {description}/>
+    </div>
+  );
+}`;
```

Comment:
please lint the code (check other files as well)

Reply:
done. I am using `pnpm lint fix` to do that, is it okay?